metadatastore/core.py

```diff
@@ -374,6 +376,10 @@ def get_events_generator(descriptor, event_col, descriptor_col,
             if convert_arrays:
                 if _dk['dtype'] == 'array' and not _dk.get('external', False):
                     ev['data'][k] = np.asarray(ev['data'][k])
+
+        # note which keys refer to dereferences (external) data
+        ev['filled'] = filled
```

Comment:
Did this ever get fixed?  It looks like the same `filled` object is still shared across all of the events.